clippy_lints/src/consts.rs

```diff
@@ -153,8 +153,6 @@ impl Constant {
 
 /// Parses a `LitKind` to a `Constant`.
 pub fn lit_to_constant(lit: &LitKind, ty: Option<Ty<'_>>) -> Constant {
-    use syntax::ast::*;
```

Comment:
This produced one `unused_imports` warning, since `FloatTy` and `LitKind` was imported twice after fixing it.

Reply:
It might be worth adding this to the `Known Problems` section?